src/components/number/number.test.js

```diff
@@ -346,4 +333,169 @@ context("Tests for Number component", () => {
       }
     );
   });
+
+  describe("check accessibility for Number component", () => {
```

Comment:
isn't it easier to import just all stories from `number.stories.tsx` file? and just check it?